CANTroller2/src/devices.h

```diff
@@ -408,6 +472,30 @@ class AnalogSensor : public Sensor<NATIVE_T, HUMAN_T> {
         
 };
 
+// BatterySensor reads the voltage level from the Mule battery
+class BatterySensor : public AnalogSensor<int32_t, float> {
+    protected:
+        // NOTE: for now lets keep all the config stuff here in the class. could also read in values from a config file at some point.
+        static constexpr float _initial_adc = adcmidscale_adc;
+        static constexpr float _initial_v = 10.0;
+        static constexpr float _min_v = 0.0; // NOTE: this would be if the battery is totally dead...? (not that we'd be able to run this code anyway...)
```

Comment:
What does this comment mean?

Reply:
I'll remove it, I was mostly just me wondering why we even have a minimum, since if we had no battery voltage we wouldn't be running...

but i'll remove it, np